pkg/skaffold/build/tag/git_commit_test.go

```diff
@@ -354,13 +333,46 @@ func TestGitCommit_GenerateFullyQualifiedImageName(t *testing.T) {
 				t.CheckNoError(err)
 
 				tag, err := tagger.GenerateFullyQualifiedImageName(workspace, "test")
-				t.CheckNoError(err)
-				t.CheckDeepEqual(expectedTag, tag)
+
+				t.CheckErrorAndDeepEqual(test.shouldErr, err, expectedTag, tag)
 			}
 		})
 	}
 }
 
+func TestGitCommitSubDirectory(t *testing.T) {
```

Comment:
Was there a reason why you had to pull this out into a separate test?  It looks the same to me…

Reply:
It's not exactly the same :-) Now it fails for some of the cases